addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -360,14 +360,24 @@
         <t t-set="go_back_url" t-valuef="/forum/#{ slug(forum) }/"/>
     </t>
     <t t-else="" t-set="_no_results_title">Oops!</t>
+    <t t-if="filters == 'unanswered'" t-set="filtered_results">
+        Sorry, we could not find any <b>unanswered</b> results
+    </t>
+    <t t-elif="filters == 'solved'" t-set="filtered_results">
+        Sorry, we could not find any <b>solved</b> results
+    </t>
+    <t t-elif="filters == 'unsolved'" t-set="filtered_results">
+        Sorry, we could not find any <b>unsolved</b> results
+    </t>
     <div t-attf-class="#{ 'o_caught_up_alert' if queue_type else '' } row g-0 justify-content-center #{ 'd-none' if hide_alert else '' }">
         <div class="p-3 d-flex flex-column align-items-center">
             <img t-if="queue_type" src="/website_forum/static/src/img/tasks.svg" class="img img-fluid mb-3" width="200" alt="Animation of a pen checking a checkbox"/>
             <img t-else="" src="/website_forum/static/src/img/empty.svg" class="img img-fluid mb-3" width="150" alt="Empty box"/>
             <h5 t-out="_no_results_title"/>
             <span t-if="result_msg" t-out="result_msg"/>
             <span t-else="">
-                Sorry, we could not find any <b t-if="filters in ['unanswered', 'solved', 'unsolved']" t-out="filters"/> results
+                <t t-if="filtered_results" t-out="filtered_results"/>
```

Comment:
Why not computing filtered_results completely here above with a t-else ? That way you don't have a first if-else nest, then another one ?

Reply:
Because ... I was wrong?